### ISSUE TITLE
.git folder accessible on website

### ISSUE THREAD

**Issue by ShaddyDC** [rebane2001/lyra.horse#1]
Currently, nginx is configured to allow access to the .git folder on your website: https://lyra.horse/.git/config

If you've got any secrets or something in there, you may risk exposing them, but with a public repo, the risk is probably pretty low anyway. I figured I'll let you know regardless in case it's something you want to fix.

**Comment by rebane2001** [comment]
Yep, it's not authenticated so you can get the same git folder yourself by just cloning this repo here

**Comment by ShaddyDC** [comment]
I figured, feel free to close then if it's no problem, and thanks for the quick response!
